### PR TITLE
fix parsing table names with a leading '.'

### PR DESCRIPTION
**Pull request by osheroff**
addresses https://github.com/zendesk/maxwell/issues/272

### CODE REVIEW

**Comment by osheroff** [comment]
@zendesk/rules @nmaquet

File: src/main/antlr4/imports/mysql_idents.g4 [hunk 1]
```diff
@@ -4,6 +4,7 @@ import mysql_literal_tokens;
 
 db_name: name;
 table_name: (db_name '.' name)
+            | '.' name // *shakes head in sadness*
```
Comment (nmaquet) [inline src/main/antlr4/imports/mysql_idents.g4]: :joy:

Comment (osheroff) [inline src/main/antlr4/imports/mysql_idents.g4]: the sins of our father-engineers will snowball into massive technical debts one day.  If they haven't already.

**Comment by vanchi-zendesk** [comment]
:+1:

**Pull request merged**